access/http/client.go

```diff
@@ -57,31 +57,31 @@ func (c *Client) GetBlockByID(ctx context.Context, blockID flow.Identifier) (*fl
 	return c.httpClient.GetBlockByID(ctx, blockID)
 }
 
-func (c *Client) GetLatestBlockHeader(ctx context.Context, isSealed bool) (*flow.BlockHeader, error) {
+func (c *Client) GetLatestBlockHeader(ctx context.Context, isSealed bool) (*flow.BlockHeader, flow.BlockStatus, error) {
```

Comment:
Having status being part of header it won't require changing return values, which is cleaner.